packages/x-codemod/README.md

```diff
@@ -86,9 +86,29 @@ npx @mui/x-codemod v7.0.0/pickers/preset-safe <path|folder>
 
 The list includes these transformers
 
+- [`rename-components-to-slots-pickers`](#rename-components-to-slots-pickers)
 - [`rename-default-calendar-month-to-reference-date`](#rename-default-calendar-month-to-reference-date)
 - [`rename-day-picker-classes`](/rename-day-picker-classes)
 
+#### `rename-components-to-slots-pickers`
+
+Renames the `components` and `componentsProps` props to `slots` and `slotProps`, respectively.
+
+This change only affects pickers components.
+
+```diff
+ <DatePicker
+-  components={{ Toolbar: CustomToolbar }}
++  slots={{ toolbar: CustomToolbar }}
+-  componentsProps={{ actionBar: { actions: ['clear'] } }}
++  slotProps={{ actionBar: { actions: ['clear'] } }}
+ />;
+```
+
+```bash
+npx @mui/x-codemod v6.0.0/pickers/rename-components-to-slots <path>
```

Comment:
```suggestion
npx @mui/x-codemod v7.0.0/pickers/rename-components-to-slots <path>
```

Reply:
We do not seem consistent with providing the example of running the individual codemod in every case. 🙈 
WDYT about updating the README to always include an example of running the codemod? 🤔

Reply:
Done :+1: